packages/tab-list/src/tab-list.ts

```diff
@@ -240,9 +255,8 @@ export class TabList extends Focusable {
             return;
         }
         await Promise.all([
-            await selectedElement.updateComplete,
-            await ((document as unknown) as { fonts: { ready: Promise<void> } })
-                .fonts.ready,
+            selectedElement.updateComplete,
+            document.fonts ? document.fonts.ready : Promise.resolve(),
```

Comment:
Fonts might be right on pre-chromium edge, but I can live with that. We just want Edge to be functional.